.github/workflows/update-nextcloud-openapi.yml

```diff
@@ -32,28 +32,34 @@ jobs:
           repository: nextcloud/server
           ref: ${{ matrix.server-versions }}
 
-      - name: Checkout app
+      - id: checkout
         uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
         with:
+          persist-credentials: false
           path: apps/${{ env.APP_NAME }}
           ref: ${{ matrix.branches }}
+        continue-on-error: true
```

Comment:
Instead remove branch master from initial list?

Reply:
Also should work

Reply:
Ah, now I see. We do the same on update-nextcloud-ocp, so it can be upstreamed and used by either apps.